README.md

```diff
@@ -1,2 +1,376 @@
-# framespace-ref
-FrameSpace Reference Server
+# FrameSpace Reference Server
+
+Reference server implementation for FrameSpace using Python Flask, MongoDB, and Protobufs. See [here](https://github.com/ohsu-computational-biology/ccc_api/tree/master/proto/framespace) for more information on FrameSpace.
+
+## Spin up with docker-compose
+
+Use docker-compose to spin up the FrameSpace reference server, assuming usage with docker-machine:
+
+1. Clone framespace-ref: `git clone https://github.com/ohsu-computational-biology/framespace-ref.git`
+
+1. `cd framespace-ref`
+
+1. `docker-compose build`
+
+1. `docker-compose up`
+
+1. To load test data, you need pandas. If you are in a virtualenv run `pip install pandas`. This is not included in server `requirements.txt` to decrease build and up time. Follow these commands to load: 
+
+```
+cd util
+python importer.py -c ../test/data/import.config -i ../test/data/tcgaLive*.tsv -H $(docker-machine ip default)
+```
```

Comment:
@bwalsh can you try what I suggest above and let me know. I've got it working here, and I've updated the readme.